.github/build.in.yml

```
@@ -141,6 +141,64 @@ jobs:
           ls .lake/build/bin/check-yaml
           ls .lake/packages/importGraph/.lake/build/bin/graph
 
+      - name: Check for changes to tool source files
+        uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 # v3.0.2
+        id: tool_changes
+        with:
```

Not sure if we also need the `base:` config option here:
```suggestion
        with:
          working-directory: 'pr-branch'
```